kubectl-plugin/config/migrate.go

```diff
@@ -55,7 +55,9 @@ func (o *migrateOption) runE(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	var password string
-	if password, err = o.getDevOpsPassword(); err == nil {
+	if password, err = o.getDevOpsPassword(); password == "" {
+		err = fmt.Errorf("the password of Jenkins is empty")
```

Comment:
This err assignment will cover the error returned by `o.getDevOpsPassword()`.

I suggest put previous `err` into this `err`.

Reply:
Good catch! See https://github.com/kubesphere-sigs/ks/pull/241/commits/051f86208348e2da1c2972232bcd41a6a7b2e14c